src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import ApiExplorerFeatures from '..';
+import { render, screen } from '@testing-library/react';
+import useAuthContext from '@site/src/hooks/useAuthContext';
+import { IAuthContext } from '@site/src/contexts/auth/auth.context';
+
+jest.mock('@docusaurus/router', () => ({
+  useLocation: () => ({
+    pathname: '/api-explorer',
+    hash: '',
+  }),
+  useHistory: () => ({
+    push: jest.fn(),
+  }),
+}));
+
+jest.mock('@site/src/hooks/useAuthContext');
+
+const mockUseAuthContext = useAuthContext as jest.MockedFunction<() => Partial<IAuthContext>>;
+
+describe('ApiExplorerFeatures', () => {
```

Comment:
please add these test cases as well: 

- what if user is logged out? 
- test case for pressing the button and websocket messages, api calls ,etc. 
- test case for changing the value of the dropdown
- test case for changing the payload and pressing the button

Reply:
These test cases arfe laready being writen for the indiviual components I don't think 
why do we need them for the wrapper as well